index.js

```diff
@@ -553,6 +553,10 @@ PSPDFKitView.propTypes = {
    * Controls whether or not the document will be automatically saved. Defaults to automatically saving (false).
    */
   disableAutomaticSaving: PropTypes.bool,
+  /**
+   * Controls whether or not the document's forms are editable. Defaults to forms being editable (true).
+   */
+  enableFormEditing: PropTypes.bool,
```

Comment:
Disabling this on iOS not only disables form **editing**, but also form viewing. Meaning that all forms (including their set value or contents) are essentially hidden.
Is this the desired behavior and does this match the Android behavior?

Reply:
Good point! I'll update the iOS implementation if needed to match the Android behaviour.

Reply:
On Android this only disables form editing. Widget annotations are still rendered. 

The only way I know of to disable form rendering is when the acroforms license is missing.

Reply:
I think the Android implementation makes more sense, and also matches what I would expect from `enableFormEditing`.

@radazzouz On iOS we could use `editableAnnotationTypes` to exclude widgets to get the same behavior I think.

Reply:
Thank you both for the info. I just pushed the changes in https://github.com/PSPDFKit/react-native/pull/365/commits/4da57557eac2c2d04aef9e4c1cbc182cf465ead3.